common/changes/@boostercloud/framework-core/update_dependencies_2023-07-24-09-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@boostercloud/framework-core",
+      "comment": "Update booster dependencies",
+      "type": "minor"
```

Comment:
Do we need to perform a release that is just upgrading dependencies? Unless these dependencies were upgraded for security reasons, I guess we can set this to "none" and get these upgraded dependencies on the next release along with some features 🤔

Reply:
It's a security update